Makefile

```diff
@@ -22,8 +23,12 @@ fix_format:
 	script/docs_fix_format
 	go fmt $(shell glide nv)
 
+.PHONY: check_format
+check_format: docs_format
+	@gofmt -l $(shell glide nv | sed 's/\.\.\.//g') | tee /dev/stderr | read && echo "Files failed gofmt" && exit 1 || true
+
 .PHONY: compile
-compile: docs_format
```

Comment:
not needed, that was legacy. I think that putting that in the check_format target is better, and makes more sense.